docker-compose.yaml

```diff
@@ -0,0 +1,117 @@
+version: "3.9"
+
+services:
+  mysql:
```

Comment:
we have this on prisma package, do we need to have all together in one file ?

Reply:
yep, ideally we want to spin up everything with one command.